app/util/analytics.py

```diff
@@ -105,35 +65,9 @@ def get_events(since_timestamp=None, since_id=None):
     :return: The list of events in the given range
     :rtype: list[dict] | None
     """
-    # eventlogs to STDOUT should only be used in debug situations, so don't worry about making events available.
-    if _eventlog_file is None or _eventlog_file.upper() == 'STDOUT':
+    if isinstance(_event_log, EventLog):
```

Comment:
What do you think of doing `if _event_log:` here just like up above on line 52? Were you worried about something specific with `_event_log` being some other object?